ui/pages/Project/components/PhenotypePrioritizedGenes.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+
+import { getGenesById, getPhenotypeGeneScoresByIndividual } from 'redux/selectors'
+import DataLoader from 'shared/components/DataLoader'
+import { GeneSearchLink } from 'shared/components/buttons/SearchResultsLink'
+import DataTable from 'shared/components/table/DataTable'
+import { BaseVariantGene } from 'shared/components/panel/variants/VariantGene'
+import { camelcaseToTitlecase } from 'shared/utils/stringUtils'
+import { loadPhenotypeGeneScores } from '../reducers'
+import { getPhenotypeDataLoading } from '../selectors'
+
+const PHENOTYPE_GENE_INFO_COLUMNS = [
+  {
+    name: 'geneId',
+    width: 7,
+    content: 'Gene',
+    format: ({ geneId, gene, variant, familyGuid }) => (
+      <div>
+        <BaseVariantGene
+          geneId={geneId}
+          gene={gene}
+          variant={variant}
+          compact
+          showInlineDetails
+        />
+        <GeneSearchLink
```

Comment:
unless this is coming from `BaseVariantGene` there is no need for this, the all genes link is fine

Reply:
Okay